.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,25 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/python
+{
+	"name": "pybravia",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"image": "mcr.microsoft.com/devcontainers/python:0-3.10-bullseye",
+
+	// Features to add to the dev container. More info: https://containers.dev/features.
+	"features": {
+	},
+
+	// Configure tool-specific properties.
+	"customizations": {
+		// Configure properties specific to VS Code.
+		"vscode": {
+			"settings": {},
+			"extensions": [
+				"ms-azuretools.vscode-docker"
```

Comment:
Should we add liniters there, as like https://github.com/home-assistant-libs/aioshelly/blob/main/.devcontainer/devcontainer.json ?

Reply:
I went ahead and did so, I think the specific choice of linters and whatnot is really up to you as a more ongoing maintainer of this project.

I don't know how to invoke this btw: https://github.com/home-assistant-libs/aioshelly/blob/main/.pre-commit-config.yaml

Might be relevant